NOTICK Remove example code (#6327)
corda/corda: docs/example-code/README.md

# Code examples for the documentation

> The examples have beem removed as the documentation as been moved to its own repo (see [here](../README.md)).

Reviewer: ```suggestion
The code examples directory has been removed as the documentation has been moved to its own repo (see [here](../README.md)).
```
